aspnet/fundamentals/localization.rst

```diff
@@ -203,7 +203,16 @@ If you only pass in one of the two (``culture`` or ``ui-culture``), the query st
 CookieRequestCultureProvider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie. The sample app provides code to set the cookie for French and English cultures.
+In old versions of ASP.NET a production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie.
+The ASP.NET Core use ``.AspNetCore.Culture`` cookie by default. You have to use format ``c=%LANGCODE%|uic=%LANGCODE%``, where ``c`` is CultureInfo and ``uic`` is CultureUIInfo, for instance:
+
+c='en-UK'\|uic='en-US'
+
+You can use only one of keypairs, in this case CultureInfo and CultureUIInfo will be set to same culture, for instance:
```

Comment:
"You can use only one of keypairs" sounds awkward. It sounds as if you cannot use both together which contradicts the previous part. Suggestion: "The settings can also be configured individually, in this case ..."